src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
model Classes don't require separate tests.
Constructors/getters/setters naturally involved in other tests